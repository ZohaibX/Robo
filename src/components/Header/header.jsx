import React from "react";

const Header = () => {
  console.log("Header");
  return <h1>RoboFriends</h1>;
};

export default Header;
