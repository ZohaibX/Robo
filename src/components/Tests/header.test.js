import { shallow } from "enzyme";
import React from "react";
import Header from "../Header/header";
import renderer from "react-test-renderer";

it("expect to render Header component", () => {
  const tree = renderer.create(<Header />).toJSON();

  expect(tree).toMatchSnapshot(); // for snapshot

  expect(shallow(<Header />).length).toEqual(1);

  // expect(shallow(<Header />)).toMatchSnapshot();
});
