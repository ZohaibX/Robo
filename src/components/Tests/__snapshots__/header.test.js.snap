// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Header component 1`] = `
<h1
  id="h1"
>
  RoboFriends
</h1>
`;
